### PR TITLE
[12.0][ADD]rma_internal_transfer

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Sometimes the repair location is not the same as the receiving location and users wants to perform this action manual instead of push rules. In this case an internal transfer has to be done.

This module just adds a button that eases this task

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
where is the commit history?

**Comment by AaronHForgeFlow** [comment]
> where is the commit history?

It is a new module.

File: rma_internal_transfer/tests/test_internal_transfer.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+                move.product_id.categ_id.rma_customer_operation_id = False
+                move.product_id.rma_customer_operation_id = False
+                wizard._prepare_rma_line_from_stock_move(move)
+
+            cls.line = cls.rma_line.create(data)
+            cls.line.action_rma_to_approve()
+            cls.line.action_rma_approve()
+        return rma_id
+
+    def test_0_internal_transfer(cls):
```
Comment (MiquelRForgeFlow) [inline rma_internal_transfer/tests/test_internal_transfer.py]:
```suggestion
    def test_0_internal_transfer(self):
```

File: rma_internal_transfer/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    'license': 'LGPL-3',
+    'category': 'RMA',
+    'summary': 'Internal transfers. Made easy.',
+    'author': "Eficent, Odoo Community Association (OCA)",
+    'website': 'https://github.com/Eficent/stock-rma',
+    'depends': ['rma'],
+    'data': ['wizards/rma_internal_transfer.xml',
+             ],
+    'installable': True,
+    'auto_install': False,
```
Comment (MiquelRForgeFlow) [inline rma_internal_transfer/__manifest__.py]:
```suggestion
```

File: rma_internal_transfer/README.rst [hunk 3]
```diff
@@ -0,0 +1,47 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
```
Comment (MiquelRForgeFlow) [inline rma_internal_transfer/README.rst]:
```suggestion
.. image:: https://img.shields.io/badge/license-LGPL--3-blue.png
```

File: rma_internal_transfer/README.rst [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    :alt: License LGPL-3
```
Comment (MiquelRForgeFlow) [inline rma_internal_transfer/README.rst]:
```suggestion
   :target: https://www.gnu.org/licenses/lgpl
   :alt: License: LGPL-3
```

File: rma_internal_transfer/README.rst [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+========================
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/Eficent/stock-rma/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
```
Comment (MiquelRForgeFlow) [inline rma_internal_transfer/README.rst]:
```suggestion
help us smash it by providing detailed and welcomed feedback.
```

File: rma_internal_transfer/README.rst [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+
+Usage
+=====
+
+In the RMA from view there is a new button "Internal Transfer". The user
+just need to change the default location according its needs.
+
+Known issues and Roadmap
+========================
```
Comment (MiquelRForgeFlow) [inline rma_internal_transfer/README.rst]: delete

File: rma_internal_transfer/README.rst [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+RMA Internal Transfers
+======================
+
+Sometimes the repair location is not the same as the receiving location
+and useres wants to perform this action manual instead of putaway
+strategies. In this case an internal tranfer has to be done.
+
+This module just adds a button that eases this task
+
+
```
Comment (MiquelRForgeFlow) [inline rma_internal_transfer/README.rst:15]: delete a white line, too much space
```suggestion
```

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**